aws-sdk-java-v2-master/archetypes/archetype-app-quickstart/src/main/resources/archetype-resources/src/main/java/Handler.java

```diff
@@ -8,8 +8,8 @@
     private final ${serviceClientClassName} ${serviceClientVariable}Client;
 
     public Handler() {
-        ${serviceClientVariable}Client = DependencyFactory.${serviceClientVariable}Client();
-    }
+        ${serviceClientVariable}Client = DependencyFactory.${serviceClientVariable}Client()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;;&#34; to complete Statement